scripts/deploy-backend.sh

```diff
@@ -161,13 +185,13 @@ bind_backend_to_services() {
 # Determine the appropriate BASE_URL for the deployed instance based on the
 # provided Cloud.gov App Name
 DEFAULT_ROUTE="https://$CGAPPNAME_FRONTEND.app.cloud.gov"
-if [ -n "$BASE_URL" ]; then
+if [[ -n $BASE_URL ]]; then
   # Use Shell Parameter Expansion to replace localhost in the URL
   BASE_URL="${BASE_URL//http:\/\/localhost:8080/$DEFAULT_ROUTE}"
-elif [ "$CF_SPACE" = "tanf-prod" ]; then
+elif [[ $CF_SPACE == 'tanf-prod' ]]; then
```

Comment:
I just brought in line to be like everything else... some places we were using single equals and some places double... double is preferred for bash scripting so that's what I went with. there shouldn't be any single quotes now and they all should look the same

Reply:
but yeah... functionally the same fo our uses